lib/recurly/risk/three-d-secure/strategy/stripe.js

```diff
@@ -44,7 +44,8 @@ export default class StripeStrategy extends ThreeDSecureStrategy {
         if (result.error) {
           return this.threeDSecure.error('3ds-auth-error', { cause: result.error });
         }
-        this.emit('done', result.paymentIntent);
+        const { id } = result.paymentIntent;
+        this.emit('done', { id });
```

Comment:
Here's the relevant update for plucking only the `id` from the `PaymentIntent` result